### DISCUSSION TITLE
Change current user password

### DISCUSSION THREAD

**Discussion by kuatkh** [dsbenghe/Novell.Directory.Ldap.NETStandard#193] [Q&A]
hello. firstly thank you for cool package.
i have one question. why i can not change current user password? i connect to AD then bind to AD with current user credentials.
AD returns that i have not permissions to change password, when i call Modify. could you help me with this?
here is my code

                            var ldapConn = new LdapConnection();
                            ldapConn.SecureSocketLayer = true;
                            ldapConn.UserDefinedServerCertValidationDelegate += (sender, certificate, chain, sslPolicyErrors) => true;
                            ldapConn.Connect(ad_Url, LdapConnection.DefaultSslPort);
                            // credentials in db
                            ldapConn.Bind(LdapConnection.LdapV3, current_user_login, current_user_password);

                            ldapConn.Modify(dn,
                                new LdapModification(
                                        LdapModification.Replace,
                                        new LdapAttribute("unicodePwd", Encoding.Unicode.GetBytes($"\"{current_user_new_password}\""))
                                    )
                                );

                            ldapConn.Disconnect();

**Comment by AltaySarsengali** [comment]
Hello! Stucked with the same problem, have you resolved it?
